huaweicloud/services/acceptance/fgs/resource_huaweicloud_fgs_function_test.go

```diff
@@ -262,62 +223,3 @@ resource "huaweicloud_fgs_function" "test" {
 }
 `, rName, acceptance.HW_ENTERPRISE_PROJECT_ID_TEST)
 }
-
-func testAccFgsV2Function_agency(rName string) string {
-	return fmt.Sprintf(`
-resource "huaweicloud_vpc" "test" {
-  name = "%s"
-  cidr = "192.168.0.0/16"
-}
-
-resource "huaweicloud_vpc_subnet" "test" {
-  name       = "%s"
-  cidr       = "192.168.1.0/24"
-  gateway_ip = "192.168.1.1"
-  vpc_id     = huaweicloud_vpc.test.id
-}
-
-resource "huaweicloud_sfs_file_system" "test" {
-  share_proto = "NFS"
-  size        = 10
-  name        = "%s"
-  description = "test sfs for fgs"
-}
-
-resource "huaweicloud_identity_agency" "test" {
-  name                   = "%s"
-  description            = "test agency for fgs"
-  delegated_service_name = "op_svc_cff"
```

Comment:
shift to an env variable of remove it ?

Reply:
the conclusion of morning meeting is to remove it, if you want another way, please rediscuss with niuzhenguo